<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LobbyWindow</class>
 <widget class="QMainWindow" name="LobbyWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1440</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Diceobu</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/img/d20v2.png</normaloff>:/img/d20v2.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>770</width>
      <height>770</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTableWidget{
font: 75 30pt &quot;ImperatorSmallCaps&quot;;
background-color: gray;
border-style: solid;
border-color: black;
border-width: 5px;
border-radius: 10px;
color: white;
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>10</y>
      <width>761</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 black, stop:1 gray);
font: 75 30pt &quot;ImperatorSmallCaps&quot;;
border-style: solid;
border-color: black;
border-width: 2px;
border-radius: 10px;
color: white;
}</string>
    </property>
    <property name="text">
     <string>Game Map</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>10</y>
      <width>182</width>
      <height>222</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <property name="spacing">
      <number>20</number>
     </property>
     <item>
      <widget class="QPushButton" name="pushButton_Character_Details">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>180</width>
         <height>60</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 white, stop:1 gray);
font: 75 18pt &quot;ImperatorSmallCaps&quot;;
border-style: solid;
border-color: black;
border-width: 2px;
border-radius: 10px;
color: black;
}


QPushButton:pressed{
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 white, stop:1 grey);
font: 75 22pt &quot;ImperatorSmallCaps&quot;;
border-color: black;
border-width: 1px;
border-radius: 10px;
}
</string>
       </property>
       <property name="text">
        <string>Character
Details</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>180</width>
         <height>60</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 white, stop:1 gray);
font: 75 24pt &quot;ImperatorSmallCaps&quot;;
border-style: solid;
border-color: black;
border-width: 2px;
border-radius: 10px;
color: black;
}


QPushButton:pressed{
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 white, stop:1 grey);
border-color: black;
border-width: 1px;
border-radius: 10px;
}
</string>
       </property>
       <property name="text">
        <string>Spells</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>1170</x>
      <y>10</y>
      <width>251</width>
      <height>631</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox{
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0  rgba(128,128, 128), stop:1 rgba(64,64, 64));
font: 75 24pt &quot;ImperatorSmallCaps&quot;;
border-style: solid;
border-color: black;
border-width: 2px;
border-radius: 10px;
color: black;
}</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>251</width>
       <height>60</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 black, stop:1 gray);
font: 75 30pt &quot;ImperatorSmallCaps&quot;;
border-style: solid;
border-color: black;
border-width: 2px;
border-radius: 10px;
color: white;
}</string>
     </property>
     <property name="text">
      <string>Friends</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_Move">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>270</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 white, stop:1 gray);
font: 75 12pt &quot;ImperatorSmallCaps&quot;;
border-style: solid;
border-color: black;
border-width: 2px;
border-radius: 10px;
color: black;
}


QPushButton:pressed{
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 white, stop:1 grey);
border-color: black;
border-width: 1px;
border-radius: 10px;
}
</string>
    </property>
    <property name="text">
     <string>Move
Character</string>
    </property>
   </widget>
   <widget class="MapWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>751</width>
      <height>750</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTableWidget{
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 gray, stop:1 black);
font: 75 30pt &quot;ImperatorSmallCaps&quot;;
border-style: solid;
border-color: black;
border-width: 2px;
border-radius: 10px;
color: white;
}</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>250</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Not paint</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>801</x>
      <y>650</y>
      <width>621</width>
      <height>221</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="minimumSize">
        <size>
         <width>310</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>310</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>ImperatorSmallCaps</family>
         <pointsize>16</pointsize>
         <weight>9</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 white, stop:1 gray);
font: 75 16pt &quot;ImperatorSmallCaps&quot;;
border-style: solid;
border-color: black;
border-width: 2px;
border-radius: 10px;
color: black;
}</string>
       </property>
       <property name="text">
        <string>System Log</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
       <property name="buddy">
        <cstring>system_log</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextBrowser" name="system_log">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QTextBrowser{
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 gray, stop:1 gray);
font: 75 11pt &quot;ImperatorSmallCaps&quot;;
border-style: solid;
border-color: black;
border-width: 2px;
border-radius: 10px;
color: black;
}</string>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'ImperatorSmallCaps'; font-size:11pt; font-weight:72; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>430</y>
      <width>275</width>
      <height>168</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="minimumSize">
          <size>
           <width>129</width>
           <height>60</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>130</width>
           <height>60</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel{
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 black, stop:1 gray);
font: 75 14pt &quot;ImperatorSmallCaps&quot;;
border-style: solid;
border-color: black;
border-width: 2px;
border-radius: 10px;
color: white;
}</string>
         </property>
         <property name="text">
          <string>Current 
 Map</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_currMap">
         <property name="minimumSize">
          <size>
           <width>129</width>
           <height>60</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>130</width>
           <height>60</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>ImperatorSmallCaps</family>
           <pointsize>14</pointsize>
           <weight>9</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel{
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 white, stop:1 gray);
font: 75 14pt &quot;ImperatorSmallCaps&quot;;
border-style: solid;
border-color: black;
border-width: 2px;
border-radius: 10px;
color: black;
}

</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <spacer name="horizontalSpacer_7">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>15</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_nextMap">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>99</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>30</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
background-color: rgb(50,50,50);
font: 75 14pt &quot;ImperatorSmallCaps&quot;;
border-style: solid;
border-color: black;
border-width: 2px;
border-radius:5px;
color: white;
}

QPushButton:pressed{
background-color: rgb(24,24,24);
font: 75 12pt &quot;ImperatorSmallCaps&quot;;
border-style: solid;
border-color: black;
border-width: 1px;
border-radius:5px;
color: white;
}</string>
           </property>
           <property name="text">
            <string>&gt;&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_6">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>15</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="minimumSize">
          <size>
           <width>130</width>
           <height>60</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>130</width>
           <height>60</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel{
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 black, stop:1 gray);
font: 75 12pt &quot;ImperatorSmallCaps&quot;;
border-style: solid;
border-color: black;
border-width: 2px;
border-radius: 10px;
color: white;
}</string>
         </property>
         <property name="text">
          <string>Current 
 Character</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_currChar">
         <property name="minimumSize">
          <size>
           <width>130</width>
           <height>60</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>130</width>
           <height>60</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>ImperatorSmallCaps</family>
           <pointsize>14</pointsize>
           <weight>9</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel{
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 white, stop:1 gray);
font: 75 14pt &quot;ImperatorSmallCaps&quot;;
border-style: solid;
border-color: black;
border-width: 2px;
border-radius: 10px;
color: black;
}

</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>15</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_nextCharacter">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>30</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
background-color: rgb(50,50,50);
font: 75 14pt &quot;ImperatorSmallCaps&quot;;
border-style: solid;
border-color: black;
border-width: 2px;
border-radius:5px;
color: white;
}

QPushButton:pressed{
background-color: rgb(24,24,24);
font: 75 12pt &quot;ImperatorSmallCaps&quot;;
border-style: solid;
border-color: black;
border-width: 1px;
border-radius:5px;
color: white;
}</string>
           </property>
           <property name="text">
            <string>&gt;&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>15</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QComboBox" name="comboBox_Maps">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>350</y>
      <width>271</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox{
background-color: rgb(45, 45, 45);
font: 75 12pt &quot;ImperatorSmallCaps&quot;;
border-style: solid;
border-color: rgb(88, 88, 88);
border-width: 1px;
border-radius: 2px;
color: rgb(220, 220, 220);

}

</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_Characters">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>390</y>
      <width>271</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox{
background-color: rgb(45, 45, 45);
font: 75 12pt &quot;ImperatorSmallCaps&quot;;
border-style: solid;
border-color: rgb(88, 88, 88);
border-width: 1px;
border-radius: 2px;
color: rgb(220, 220, 220);

}

</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Grid">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>840</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 white, stop:1 gray);
font: 75 10pt &quot;ImperatorSmallCaps&quot;;
border-style: solid;
border-color: black;
border-width: 2px;
border-radius: 10px;
color: black;
}


QPushButton:checked{
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 rgb(170,170,170), stop:1 grey);
font: 75 10pt &quot;ImperatorSmallCaps&quot;;
border-color: black;
border-width: 1px;
border-radius: 10px;
}
</string>
    </property>
    <property name="text">
     <string>Grid</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>layoutWidget</zorder>
   <zorder>label</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>groupBox</zorder>
   <zorder>pushButton_Move</zorder>
   <zorder>tableWidget_2</zorder>
   <zorder>tableWidget</zorder>
   <zorder>pushButton_5</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>comboBox_Maps</zorder>
   <zorder>comboBox_Characters</zorder>
   <zorder>pushButton_Grid</zorder>
   <zorder>horizontalSpacer</zorder>
   <zorder>horizontalSpacer_2</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1440</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>ImperatorSmallCaps</family>
     <pointsize>11</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QMenuBar{
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 white, stop:1 gray);
font: 75 11pt &quot;ImperatorSmallCaps&quot;;
}</string>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="font">
     <font>
      <family>ImperatorSmallCaps</family>
      <pointsize>14</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QMenu{
font: 75 14pt &quot;ImperatorSmallCaps&quot;;
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 white, stop:1 gray);

}
QAction{
font: 75 14pt &quot;ImperatorSmallCaps&quot;;
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 white, stop:1 gray);

}</string>
    </property>
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuOpen">
     <property name="title">
      <string>Open...</string>
     </property>
    </widget>
    <widget class="QMenu" name="menuNew">
     <property name="font">
      <font>
       <family>ImperatorSmallCaps</family>
       <pointsize>11</pointsize>
       <weight>9</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QMenu{
font: 75 11pt &quot;ImperatorSmallCaps&quot;;
}

QAction{
font: 75 11pt &quot;ImperatorSmallCaps&quot;;
}</string>
     </property>
     <property name="title">
      <string>New...</string>
     </property>
     <addaction name="actionNewMap"/>
     <addaction name="actionNewCharacter"/>
    </widget>
    <widget class="QMenu" name="menuClose">
     <property name="title">
      <string>Close</string>
     </property>
    </widget>
    <widget class="QMenu" name="menuDelete">
     <property name="title">
      <string>Delete...</string>
     </property>
     <addaction name="actionDeleteMap"/>
     <addaction name="actionDeleteCharacter"/>
    </widget>
    <addaction name="menuOpen"/>
    <addaction name="menuNew"/>
    <addaction name="actionSave"/>
    <addaction name="menuDelete"/>
    <addaction name="menuClose"/>
    <addaction name="separator"/>
    <addaction name="actionOptions"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/img/diskette.png</normaloff>:/img/diskette.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="font">
    <font>
     <family>ImperatorSmallCaps</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </action>
  <action name="actionOptions">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/fonts/img/settings.png</normaloff>:/fonts/img/settings.png</iconset>
   </property>
   <property name="text">
    <string>Options</string>
   </property>
   <property name="font">
    <font>
     <family>ImperatorSmallCaps</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </action>
  <action name="actionNewCharacter">
   <property name="text">
    <string>Character</string>
   </property>
   <property name="toolTip">
    <string>newCharacter</string>
   </property>
   <property name="font">
    <font>
     <family>ImperatorSmallCaps</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="shortcut">
    <string>C</string>
   </property>
  </action>
  <action name="actionDeleteCharacter">
   <property name="text">
    <string>Character</string>
   </property>
   <property name="toolTip">
    <string>deleteCharacter</string>
   </property>
  </action>
  <action name="actionDeleteMap">
   <property name="text">
    <string>Map</string>
   </property>
   <property name="toolTip">
    <string>deleteMap</string>
   </property>
  </action>
  <action name="actionNewMap">
   <property name="text">
    <string>Map</string>
   </property>
   <property name="toolTip">
    <string>newMap</string>
   </property>
   <property name="font">
    <font>
     <family>ImperatorSmallCaps</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="shortcut">
    <string>M</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MapWidget</class>
   <extends>QTableWidget</extends>
   <header>mapwidget.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tableWidget</tabstop>
  <tabstop>pushButton_2</tabstop>
  <tabstop>pushButton_Move</tabstop>
  <tabstop>pushButton_5</tabstop>
  <tabstop>tableWidget_2</tabstop>
  <tabstop>pushButton_nextMap</tabstop>
  <tabstop>pushButton_nextCharacter</tabstop>
  <tabstop>comboBox_Maps</tabstop>
  <tabstop>comboBox_Characters</tabstop>
  <tabstop>pushButton_Grid</tabstop>
 </tabstops>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
